resources/js/components/Kiosk.vue

```diff
@@ -559,18 +554,30 @@ export default {
                 );
             }
         },
-        axiosErrorToBugsnag(error) {
-            Bugsnag.notify(error, function (event) {
+        axiosErrorToSentry(error) {
+            if (process.env.MIX_SENTRY_DSN !== undefined) {
+                let axiosContext = null
                 if (error.response) {
-                    event.addMetadata('axios-response', error.response)
-                    event.request.url = error.response.config.url
-                    event.request.headers = error.response.config.headers
-                    event.request.httpMethod = error.response.config.method
-                    event.request.body = error.response.config.data
+                    axiosContext = {
+                        'Response Status': error.response.status,
+                        'Response Body': error.response.data,
+                        'Request URL': error.response.config.url,
+                        // Disabled sending headers until we can sanitize API credentials
+                        // 'request_headers': error.response.config.headers,
```

Comment:
You might be able to set Sentry to scrub these automatically

Reply:
Will investigate that in a subsequent PR